src/Framework/MockObject/MockBuilder.php

```diff
@@ -178,6 +178,31 @@ public function setMethods(array $methods = null): self
         return $this;
     }
 
+    /**
+     * Specifies the subset of methods to mock, requiring each to exist in the class
+     */
+    public function setRealMethods(array $methods = null): self
```

Comment:
Can we add an `@param string[] $methods` annotation to help static analysis?

Reply:
Thanks, added!